test/Microsoft.Azure.Devices.Edge.Test.Common/Platform.cs

```diff
@@ -1,12 +1,14 @@
 // Copyright (c) Microsoft. All rights reserved.
 namespace Microsoft.Azure.Devices.Edge.Test.Common
 {
+    using System;
     using System.Runtime.InteropServices;
+    using Microsoft.Azure.Devices.Edge.Util;
 
     public static class Platform
     {
-        public static IEdgeDaemon CreateEdgeDaemon(string installerPath) => IsWindows()
-            ? new Windows.EdgeDaemon(installerPath)
+        public static IEdgeDaemon CreateEdgeDaemon(Option<string> installerPath) => IsWindows()
+            ? new Windows.EdgeDaemon(installerPath.Expect(() => new ArgumentException()))
```

Comment:
if we want to install release package on Windows, should installerPath be empty? are you going to handle this case in separate PR? if so, add a comment.

Reply:
Yes I plan to add that later. I'll add a comment

Reply:
Added.